include/openssl/ssl.h

```diff
@@ -1435,6 +1441,16 @@ DECLARE_PEM_rw(SSL_SESSION, SSL_SESSION)
 # define SSL_get_max_proto_version(s) \
         SSL_ctrl(s, SSL_CTRL_GET_MAX_PROTO_VERSION, 0, NULL)
 
+/*
+ * Backwards compatibility, original 1.1.0 names
+ *
+ * TODO: backport to 1.1.0, ensuring the new API works in both.
```

Comment:
I don't think this TODO belongs in the master/1.1.1 source - since that means we need to update those branches to remove this comment when doing the backport to 1.1.0 - which doesn't make any sense to me.

Reply:
I'll remove the TODO comment.